protos/protos/public/modeldb/metadata/MetadataService.proto

```diff
@@ -90,6 +90,37 @@ message DeletePropertyRequest {
     }
 }
 
+message AddKeyValuePropertiesRequest {
+    IdentificationType id = 1;
+    string property_name = 2;
+    repeated common.KeyValue key_value = 3; // Only string value expected
```

Comment:
split this up into key and value both of which are string, to avoid this confiusion

Reply:
But if I am separating key-value then how I will return it in GetAll API response within a repeated form?

Reply:
We need one message to identify key-value pairs for a repeated key-value response. if you think, instead of KeyValue I have to create a new message for key and just string value then that would be fine. what's your thought?

Reply:
message StringKeyValue {
  string key =1;
  string value =2;
} 

that would solve the problem?

Reply:
Yes.

Reply:
Changed